osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-management.xsd

```diff
@@ -28,6 +28,36 @@
     </xsd:complexType>
   </xsd:element>
 
+  <xsd:complexType name="GetOutagesRequestData">
+    <xsd:complexContent>
+      <xsd:extension base="common:Query">
+        <xsd:sequence>
+        </xsd:sequence>
+      </xsd:extension>
+    </xsd:complexContent>
+  </xsd:complexType>
+
+  <!-- Outage containing endTime and duration -->
+  <xsd:complexType name="Outage">
+    <xsd:sequence>
+      <xsd:element name="endTime" type="xsd:dateTime"
+                   minOccurs="1" maxOccurs="1"/>
```

Comment:
minOccurs="1" maxOccurs="1" can be ommitted as they are the default.

Reply:
true, removed this redundancy throughout the whole xsd, like a true boyscout